sdk/src/test/java/com/finbourne/lusid/tutorials/marketdata/Quotes.java

```diff
@@ -0,0 +1,116 @@
+package com.finbourne.lusid.tutorials.marketdata;
+
+import com.finbourne.lusid.ApiClient;
+import com.finbourne.lusid.ApiException;
+import com.finbourne.lusid.api.QuotesApi;
+import com.finbourne.lusid.model.*;
+import com.finbourne.lusid.utilities.ApiClientBuilder;
+import com.finbourne.lusid.utilities.TestDataUtilities;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasSize;
+
+public class Quotes {
+
+    private static QuotesApi quotesApi;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        ApiClient apiClient = new ApiClientBuilder("secrets.json").build();
+        quotesApi = new QuotesApi(apiClient);
+    }
+
+    @Test
+    public void add_quote() throws ApiException {
+        UpsertQuoteRequest  request = new UpsertQuoteRequest()
+            .quoteId(new QuoteId()
+                .provider("DataScope")
+                .priceSource("USDRC")
```

Comment:
Change all priceSource to be "" which is the new default.

Reply:
removed